solidity/random-beacon/test/RandomBeacon.Parameters.test.ts

```diff
@@ -44,18 +50,34 @@ describe("RandomBeacon - Parameters", () => {
     })
 
     context("when the caller is the governance", () => {
-      let tx: ContractTransaction
+      let tx1: ContractTransaction
+      let tx2: ContractTransaction
+      let tx3: ContractTransaction
 
       before(async () => {
         await createSnapshot()
 
-        tx = await randomBeacon
+        await randomBeaconGovernance
           .connect(governance)
-          .updateRelayEntryParameters(
-            newRelayEntrySoftTimeout,
-            newRelayEntryHardTimeout,
-            newCallbackGasLimit
-          )
```

Comment:
Ok, if the idea was to check the `update*` functions in isolation, then I think it makes sense to go with 1) because not all of the tests are present in the `RandomBeaconGovernance.test.ts`. Some of them actually check the reverted messages or events emitted in the `RandomBeacon`, and IMO it makes sense to leave this file. However, just doing `await ethers.getSigner(RandomBeaconGovernance.address);` didn't work and HH was complaining. For the testing purposes, I think it's fine just to use the transfer governance functions [e5b288d](https://github.com/keep-network/keep-core/pull/3018/commits/e5b288dff48bd041a8bada70eb6b78c9a40eff24)